packages/admin-client/openapi-ts.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from '@hey-api/openapi-ts';
+
+export default defineConfig({
+  input: 'http://localhost:3001/api-json',
```

Comment:
would this not be an env var?

Reply:
This is actually only run during code generation and not during something like vite build